graph-gateway/src/network/snapshot.rs

```diff
@@ -53,15 +57,17 @@ pub struct Indexing {
     /// The indexer
     pub indexer: Arc<Indexer>,
 
-    /// The indexer's indexing status
-    pub status: Option<IndexingStatus>,
+    /// The indexing indexation progress.
+    ///
+    /// See [`IndexationProgress`] for more information.
+    pub progress: IndexationProgress,
     /// The indexer's indexing cost model
     pub cost_model: Option<Ptr<CostModel>>,
 }
 
-/// The [`IndexingStatus`] struct represents the indexer's indexing status.
+/// The [`IndexationProgress`] struct represents the indexation progress of an indexing.
```

Comment:
The word "indexation" doesn't really work in this context. Maybe `IndexingProgress` or `IndexingStatus` (since it comes from the indexer's status endpoint)?

Reply:
Resolved in https://github.com/edgeandnode/gateway/pull/774